nobl9/resource_alert_policy.go

```diff
@@ -137,10 +147,11 @@ func marshalAlertPolicy(d *schema.ResourceData) (*v1alphaAlertPolicy.AlertPolicy
 			Labels:      labelsMarshaled,
 		},
 		v1alphaAlertPolicy.Spec{
-			Description:  d.Get("description").(string),
-			Severity:     d.Get("severity").(string),
-			Conditions:   marshalAlertConditions(d),
-			AlertMethods: marshalAlertMethods(d),
+			Description:      d.Get("description").(string),
+			Severity:         d.Get("severity").(string),
+			CoolDownDuration: d.Get("cool_down").(string),
```

Comment:
In [YAML](https://github.com/nobl9/nobl9-go/blob/fb4cf97ce025f114437a18684dbd0fd38738a435/manifest/v1alpha/alertpolicy/alert_policy.go#L43) we have a `coolDown` (camel case), but in Terraform we should present it with a snake case convention, so `cool_down` IMO is more fitting here? WDYT?

Reply:
https://docs.nobl9.com/yaml-guide/?YAMLs=AlertPolicy+general+sample in the documentation we show `cooldown`:
![Uploading image.png…]()

Reply:
Well. As we discussed in the office, I agree with you that cooldown is a correct word and we should follow documentation here. I changed it in terraform and maybe some day we can change it in SDK too, but for now it could be a breaking change for customers that stores SLO as a code in repositories.